tempest/api/workloadmgr/base.py

```diff
@@ -810,7 +812,8 @@ def snapshot_restore(
             workload_id,
             snapshot_id,
             restore_name="",
-            restore_cleanup=True):
+            restore_cleanup=True,
+            sec_group_cleanup=False):
```

Comment:
This need to be true by default. You can set it to False in specific scripts wherever required.

Reply:
Made changes as per review comment.